docs/instructlab-hardware-matrix.md

```diff
@@ -0,0 +1,44 @@
+|M1 Ultra |M1 |Integrated graphics |? |?|? |🟡 Supported in theory, but not regularly tested by CI|[m1_ultra.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/apple/m1/m1_ultra.yaml)
+|M2 (Standard) |M2 |Integrated graphics |? |? |? |🟡 Supported in theory, but not regularly tested by CI |[m2_yaml.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/apple/m2/m2.yaml)
+|M2 Max |M2 |Integrated graphics |? |? |? |🟡 Supported in theory, but not regularly tested by CI |[m2_max.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/apple/m2/m2_max.yaml)
+|M2 Pro |M2 |Integrated graphics |? |? |? |🟡 Supported in theory, but not regularly tested by CI |[m2_pro.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/apple/m2/m2_pro.yaml)
+|M2 Ultra |M2 |Integrated graphics |? |? |? |🟡 Supported in theory, but not regularly tested by CI |[m2_ultra.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/apple/m2/m2_ultra.yaml)
+|M3 (Standard) |M3 |Integrated graphics |? |? |? |🟡 Supported in theory, but not regularly tested by CI |[m3.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/apple/m3/m3.yaml)
+|M3 Max |M3 |Integrated graphics |? |? |? |🟡 Supported in theory, but not regularly tested by CI |[m3_max.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/apple/m3/m3_max.yaml)
+|M3 Pro |M3 |Integrated graphics |? |? |? |🟡 Supported in theory, but not regularly tested by CI |[m3_pro.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/apple/m3/m3_pro.yaml)
+
+## NVIDIA Accelerators
+
+|System Profile |CPU |GPU |RAM |Average SDG time |Average training time |CI support |System profile YAML file
+|---------------------------|-------|------|----------------|--------|---|-----------|-----------------
+|2x A100 |? |A100 |160 GB |? |? |🔴 Not supported |[a100_x2.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/nvidia/a100/a100_x2.yaml)
+|4x A100 |? |A100 |320 GB |? |? |🔴 Not supported |[a100_x4.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/nvidia/a100/a100_x4.yaml)
+|8x A100 |? |A100 |640 GB |? |? |🔴 Not supported |[a100_x4.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/nvidia/a100/a100_x8.yaml)
+|2x H100 |? |H100 |160 GB |? |? |🔴 Not supported |[h100_x2.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/nvidia/h100/h100_x2.yaml)
+|4x H100 |? |H100 |320 GB |? |? |🔴 Not supported |[h100_x4.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/nvidia/h100/h100_x4.yaml)
+|8x H100 |? |H100 |640 GB |? |? |🔴 Not supported|[h100_x4.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/nvidia/h100/h100_x8.yaml)
```

Comment:
Why are these listed as not supported? Shouldn't they be yellow?

Reply:
These were others where I wasn't sure if it was technically supported because I actually just wasn't sure if they were tested.

The other thing about these was that they're more RHEL AI use cases, and I was uncertain about how we test the downstream supported hardware in the upstream CI

But! If they are technically supported, I'll change it to the yellow!

Reply:
We don't test it explicitly upstream, but if it works downstream it should work here as well!

Reply:
👍